system-int-test/src/cmr/system_int_test/utils/bootstrap_util.clj

```diff
@@ -162,6 +162,41 @@
          body (json/decode (:body response) true)]
      (assoc body :status (:status response)))))
 
+(defn- fingerprint-by-url
```

Comment:
Here's the place where you would add the query parameters that you want to use in the tests like synchronous=true.

Reply:
I didn't do it here  because I am passing in the provider id through parameters, which is easier to invoke, but not the POST standard where the payload should be in the body. I think I am going to change the api to pass in the provider id in the request body.